src/screens/regimen_setup/currentProducts/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+import Button from '../../../components/button';
+import { useThemeContext } from '../../../theme';
+import applyScale from '../../../utils/applyScale';
+import { Container, Image, Content, Title, SubContent } from './style';
+
+interface Iprops {
+  title: string;
+}
+
+export default function CurrentProduct(props: Iprops) {
+  const { colors, fonts } = useThemeContext();
+  const [state, setstate] = useState<'flex' | 'none'>('flex');
+
+  const clickAction = () => {
+    setstate('none');
+  };
+
+  return (
+    <Container style={{ display: state }}>
+      <Image source={require('../../../../assets/images/logo.png')} />
+      <Content>
+        <Title>{props.title}</Title>
+        <SubContent>
+          <Button
+            title="&#10010; Add product"
+            activeOpacity={0}
+            buttonStyle={{
+              paddingLeft: 0,
+              paddingTop: 0,
+              paddingBottom: 0,
+              paddingRigth: 0,
+              backgroundColor: colors.BUTTON_LIGHT_COLOR
+            }}
+            textStyle={{
+              width: applyScale(98),
+              height: applyScale(20),
+              color: '#BAB200',
```

Comment:
you are suppose to move this color to the color file

Reply:
thanks boss, I've moved it to theme